templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -196,6 +196,17 @@ contents:
     # ranges are separed by comma.
     gid_mappings = ""
 
+    # The minimal amount of time in seconds to wait before issuing a timeout
+    # regarding the proper termination of the container.
+    ctr_stop_timeout = 10
```

Comment:
rpms on 1.13.9 will generate 0 here

Reply:
So we need to change the rpm to 10 then I believe.

Reply:
we don't really need it, it will be set to 10 if the value is less than 10

Reply:
Let's just skip adding this as discussed. It will default to 10 automatically.